tests/pipelines/stable_diffusion_adapter/test_stable_diffusion_adapter.py

```diff
@@ -607,175 +607,14 @@ def tearDown(self):
         gc.collect()
         torch.cuda.empty_cache()
 
-    def test_stable_diffusion_adapter_color(self):
```

Comment:
I think testing for different variants isn't too bad in general. Instead of deleting them how about we keep all of them but one (perhaps the most downloaded one; depth) commented? We could also do the same for ControlNet tests. 

WDYT?